lib/evmone/analysis.hpp

```diff
@@ -74,11 +67,11 @@ struct AdvancedExecutionState : ExecutionState
     /// Resets the contents of the execution_state so that it could be reused.
     void reset(const evmc_message& message, evmc_revision revision,
         const evmc_host_interface& host_interface, evmc_host_context* host_ctx,
-        const uint8_t* code_ptr, size_t code_size, const AdvancedCodeAnalysis& a) noexcept
+        const uint8_t* code_ptr, size_t code_size) noexcept
```

Comment:
Is `reset` needed after this?

Reply:
Reset seems better than `state = {}` because in the second case it will also copy/clear the stack data. The stack data can stay dirty.

Reply:
Silkworm is using reset() because they maintain a execution state pool.